mapper/cchainatomictx/tx_builder.go

```diff
@@ -27,12 +27,13 @@ func BuildTx(opType string, matches []*parser.Match, metadata Metadata, codec co
 	case mapper.OpExport:
 		return buildExportTx(matches, metadata, codec, avaxAssetID)
 	case mapper.OpImport:
-		return buildImportTx(matches, metadata, avaxAssetID)
+		return buildImportTx(matches, metadata, codec, avaxAssetID)
```

Comment:
We can safely remove dummy tx signing from [estimateGas function](https://github.com/ava-labs/avalanche-rosetta/blob/pchain_construction_cleanup/service/backend/cchainatomictx/construction.go#L115L118).

Reply:
Thanks for spotting this, cleaned up. Also dropped another duplicated signing.